ui/app/pages/error/error.component.js

```diff
@@ -34,12 +34,24 @@ class ErrorPage extends PureComponent {
     const { t } = this.context;
 
     const isPopup = getEnvironmentType() === ENVIRONMENT_TYPE_POPUP;
+    const messageKey = isPopup ? 'errorPagePopupMessage' : 'errorPageMessage';
 
     return (
       <section className="error-page">
         <h1 className="error-page__header">{t('errorPageTitle')}</h1>
         <h2 className="error-page__subheader">
-          {isPopup ? t('errorPagePopupMessage') : t('errorPageMessage')}
+          {t(messageKey, [
+            <a
+              target="_blank"
+              key="metamaskSupportLink"
+              rel="noopener noreferrer"
+              href="https://metamask.zendesk.com/hc/en-us/requests/new"
+            >
+              <span className="error-page__link-text">
+                {this.context.t('here')}
+              </span>
+            </a>,
+          ])}
```

Comment:
The script we use for linting has an easier time finding usages of message keys if they're used directly as string literals passed to the `t` function. See here for more about that: https://github.com/MetaMask/metamask-extension/blob/develop/development/verify-locale-strings.js#L173

So for cases like this, it would be better to assign the entire message in the conditional, factoring common parameters out as needed. e.g.

```
const supportLink = (
  <a
    target="_blank"
    key="metamaskSupportLink"
    rel="noopener noreferrer"
    href="https://metamask.zendesk.com/hc/en-us/requests/new"
  >
    <span className="error-page__link-text">
      {this.context.t('here')}
    </span>
  </a>,
);

const message = isPopup ? t('errorPagePopupMessage', [supportLink]) : t('errorPageMessage', [supportLink]);
```

Reply:
Good to know RE: the linting script. I've made the suggested changes